client/src/components/Count.tsx

```
@@ -0,0 +1,32 @@
+interface CountProps {
+  imageSrc: string;
+  title: string;
+  cnt: string;
+}
+
+const Count: React.FC<CountProps> = ({ imageSrc, title, cnt }) => {
+  return (
+    <>
+      <div className="flex flex-col items-center justify-center border border-gray-300 p-6 basis-full rounded-lg shadow-sm hover:shadow-lg bg-gray-50">
+        
+        <img src={imageSrc} alt="Card Image" className="w-auto h-24 " />
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)